src/Bicep.MSBuild.E2eTests/examples/NuGet.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+  This NuGet.config file is used to stop package referencing from nuget.org and force a local feed.
+  The intent here is to validate our local build of the MSBuild task.
+-->
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="Local" value="local-packages" />
```

Comment:
>local-packages [](start = 28, length = 14)

The GH and ADO build is setup to drop packages here when they are built, so the project will use newest package from the build.